src/sidebar/icons.js

```diff
@@ -8,10 +8,12 @@ export default {
   'arrow-left': require('../images/icons/arrow-left.svg'),
   'arrow-right': require('../images/icons/arrow-right.svg'),
   cancel: require('../images/icons/cancel.svg'),
+  'caret-left': require('../images/icons/caret-left.svg'),
```

Comment:
These are here because otherwise they won't get picked up by tests. We should chat about this!